pkg/reference/reference.go

```diff
@@ -11,6 +13,9 @@ var (
 
 	// defaultTag is latest if there is no tag
 	defaultTag = "latest"
+
+	// defaultRegistry is used to add prefix of image if needed.
+	defaultRegistry = "registry.hub.docker.com/library/"
```

Comment:
Could you remove this part in this PR? @zeppp 😄 
I am not meaning this part is not correct. I am just feel improper to combine two different parts of work into a single commit. I may list some potential shortcomings for this:

1. code committed is not atomic. So that when we revert things in the future, it will be hard.
2. It will confuse code readers on code block committed.
3. It brings more difficulties to code review.

So could you remove this in this PR and sumbit another PR for the defaultRegistry thing? I know this will bring some inconvenience for you since this pr has been delayed for a long time. While I believe "No is temporary, Yes is forever". Things we did are worthy. Thanks a lot. @zeppp